### PR TITLE
Add document section and video about generating access tokens

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Adds a section on generating tokens and video to the composite getting started page.

image

image

Checkout the chromatic storybook link to see the video in action

# Why
It is currently not clearly documented anywhere in our storybook.

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by alcail — APPROVED** [review]

**Comment by alcail** [comment]
awesome!!!
Just wondering if it will be better to add this in its own page though

**Comment by JamesBurnside** [comment]
Can we link to this also on the composite canvas page:
image

**Review by JamesBurnside — APPROVED** [review]

**Comment by anjulgarg** [comment]
> Can we link to this also on the composite canvas page: image

Any suggestion on how that can be done?

**Comment by anjulgarg** [comment]
> awesome!!! Just wondering if it will be better to add this in its own page though

Wondering the same thing, we got a lot of good feedback from our lunch and learn exercise. Let's go with this for now. We will be creating a task for moving it to its own page as a P2/P3 basis on future customer feedback.

**Comment by JamesBurnside** [comment]
> 
> 
> Any suggestion on how that can be done?

Will likely need a more creative opinion than mine, but one simple way would be:
```
To generate a token and user id follow the instructions here:
<Link url="./?path=/story/quickstarts-composites--page#generate-access-token" target="_blank">Generate composite credentials</Link>
```
(would need to check also that link would work)

**Comment by anjulgarg** [comment]
> > Any suggestion on how that can be done?
> 
> Will likely need a more creative opinion than mine, but one simple way would be below the text that already exists:
> 
> ```
> For more information on how to generate a token and user id follow the instructions here:
> <Link url="./?path=/story/quickstarts-composites--page#generate-access-token" target="_blank">Generate composite credentials</Link>
> ```
> 
> (would need to check also that link would work)
> 
> Edit: Or perhaps it could be
> 
> ```
> Please provide a <Link>access token and userID</Link>, as well as a display name in the spaces below
> ```

Smart, I completely ignored that helper text.
Added, check it out.

File: packages/storybook/stories/MeetingComposite/Utils.tsx [hunk 1]
```diff
@@ -3,14 +3,30 @@
...
 import { CompositeConnectionParamsErrMessage } from '../CompositeStringUtils';
+import { MICROSOFT_AZURE_ACCESS_TOKEN_QUICKSTART } from '../constants';
 
 export const ConfigHintBanner = (): JSX.Element => {
-  const emptyConfigTips = 'Please provide an access token, userId, endpointUrl and display name to use.';
+  const emptyConfigTips = (
+    <p>
+      Please provide an{' '}
+      <a href={MICROSOFT_AZURE_ACCESS_TOKEN_QUICKSTART} target="_blank" rel="noreferrer">
+        access token
```
Comment (JamesBurnside) [inline packages/storybook/stories/MeetingComposite/Utils.tsx:13]: nit:, user ID is also retrieved using this link

**Pull request merged**